diagnostic.md

```diff
@@ -27,7 +27,11 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$h1-font-size: ($base-font-size * 1.3);
+
+h1 {
+  font-size: $h1-font-size;
+}
```

Comment:
Correct. Though since it's pretty explicit that we'll only be using this with h1, we could also just write
```
h1 {
  font-size: ($base-font-size * 1.3);
}
```